nf_core/modules/test_yml_builder.py

```diff
@@ -258,11 +274,15 @@ def run_tests_workflow(self, command):
                     log.info(f"Setting env var '$PROFILE' to '{profile}'")
 
         tmp_dir = tempfile.mkdtemp()
+        tmp_dir_repeat = tempfile.mkdtemp()
         command += f" --outdir {tmp_dir}"
+        command_repeat = command + f" --outdir {tmp_dir_repeat}"
 
         log.info(f"Running '{self.module_name}' test with command:\n[violet]{command}")
         try:
             nfconfig_raw = subprocess.check_output(shlex.split(command))
+            nfconfig_raw = subprocess.check_output(shlex.split(command_repeat))
```

Comment:
More logs please, as these can be quite slow 👍🏻

Reply:
yes good point